.github/workflows/test.yml

```diff
@@ -39,6 +26,30 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --locked --all-features
+          args: --locked --lib
```

Comment:
Why remove --all-features? Are there no unit tests in feature gated modules we want to test?

Reply:
ack